patches/api/0466-Fix-World.getWorldType-not-detecting-amplified-and-l.patch

```diff
@@ -0,0 +1,25 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: LeonTG <leontg77@gmail.com>
+Date: Sun, 10 Mar 2024 20:52:52 +0100
+Subject: [PATCH] Fix World.getWorldType not detecting amplified and large
+ biomes
+
+
+diff --git a/src/main/java/org/bukkit/World.java b/src/main/java/org/bukkit/World.java
+index c5fe36050eeaff80cfb989fe2f38370215af6fe5..851e79257cf765a389a6c712f417fea05c453c8e 100644
+--- a/src/main/java/org/bukkit/World.java
++++ b/src/main/java/org/bukkit/World.java
+@@ -2745,12 +2745,9 @@ public interface World extends RegionAccessor, WorldInfo, PluginMessageRecipient
+      * Gets the type of this world.
+      *
+      * @return Type of this world.
+-     * @deprecated world type is only used to select the default word generation
```

Comment:
This should be left deprecated, the concept of a WorldType no longer exists, and the enum itself is even outdated because of the fact that this no longer exists.

While we can match the existing setting templates for now, this is not a mechanism which makes sense in the long run, IMHO

Reply:
Should I remove that part of the patch entirely or is it worth making it change the wording of the java doc deprecation explaination instead?

Reply:
I just thought of another idea, would methods like `isFlat()`, `isAmplified()` and `isLargeBiomes()` be a better alternative for the API portion of this?

Reply:
I disagree with keeping it deprecated. While I agree that `WorldType` is not a good representation of how worlds work now, there isn't an alternative yet and WorldType itself is not deprecated. This method was only deprecated because of the bug that this PR fixes, so un-deprecated it seems appropriate. It can be re-deprecated with a different message when alternative API is added.